src/hmem_cuda.c

```diff
@@ -376,176 +392,51 @@ static cudaError_t cuda_disabled_cudaMemcpy(void *dst, const void *src,
 	return cudaErrorInvalidValue;
 }
 
+/*
+ * Convenience macro to dynamically load symbols in cuda_ops struct
+ * Trailing semicolon is necessary for use with higher-order macro
+ */
+#define CUDA_FUNCS_DLOPEN(type, sym) \
+	do { \
+		cuda_ops.sym = dlsym(cuda_attr.type##_handle, #sym); \
+		if (!cuda_ops.sym) { \
+			FI_WARN(&core_prov, FI_LOG_CORE, "Failed to find " #sym "\n"); \
+			goto err_dlclose_cuda_driver; \
+		} \
+	} while (0);
```

Comment:
Not sure what the style guide has to say about this (@shefty can probably answer) but usually we line up the \s

Reply:
I'm not sure the guide says anything about this.  We usually do line up the '\' as it makes it a little easier to identify the define block, but I'm okay either way.

Reply:
I prefer the `\` to be aligned, but with tabs set to anything other than 8 spaces (I prefer 4), it looks pretty bad due to the varying levels of indentation in this particular macro.

Reply:
The coding style is tabs are tabs, which consist of 8 spaces.  We only need to worry about alignment around that constraint, which is also needed for indentation alignment anyway.